server/constants/vars.js

```diff
@@ -62,3 +64,49 @@ export const ALL_SUBJECTS_MONGO_PROJECTION = {
   _id: 0,
   subject: 1,
 }
+
+export const DEFAULT_CONFIG_AT_SIGNUP = (uid) => ({
```

Comment:
what do you think of something more generic like `defaultConfig`? Does it matter that this happens at signup?

```suggestion
export const defaultConfig = (uid) => ({
```

Reply:
[de48040](https://github.com/AMP-SCZ/dpdash/pull/293/commits/de48040cb5b39f723d4ad1877be815b064027992)